packages/reva-app/src/pages/TrainingProgram.tsx

```diff
@@ -0,0 +1,107 @@
+import { useActor } from "@xstate/react";
+import { FC } from "react";
+import { Interpreter } from "xstate";
+
+import { CardBasic } from "../components/atoms/CardBasic";
+import { Description } from "../components/atoms/Description";
+import { Page } from "../components/organisms/Page";
+import { MainContext, MainEvent, MainState } from "../machines/main.machine";
+
+interface Props {
+  mainService: Interpreter<MainContext, any, MainEvent, MainState, any>;
+}
+
+export const TrainingProgram: FC<Props> = ({ mainService }) => {
+  const [state] = useActor(mainService);
+
+  if (!state.context.trainingProgram) return <></>;
+
+  const {
+    trainingProgram: {
+      additionalHourCount,
+      basicSkills,
+      certificateSkills,
+      collectiveHourCount,
+      individualHourCount,
+      mandatoryTrainings,
+      otherTraining,
+    },
+    certification,
+  } = state.context;
+
+  return (
+    <Page
+      className="z-50 bg-slate-900 p-6 overflow-y-scroll"
+      direction={state.context.direction}
+    >
+      <div className="px-8 pb-8 flex flex-col">
+        <h1 className="text-white	text-3xl font-bold mb-16">
+          Votre parcours
+          <br /> personnalisé
+        </h1>
+
+        <CardBasic
+          title="Mon accompagnateur"
+          text={state.context.organism?.label}
+        />
+
+        <dl>
+          <Description term="Diplome visé">
+            {certification?.label || ""}
+          </Description>
+
+          <Description term="Nombre d'heures d'accompagnement individuel">
+            {`${individualHourCount}h`}
```

Comment:
```suggestion
            {individualHourCount}h
```

Que penses-tu de l'écrire comme cela ?